### PR TITLE
Export table html to pdf

### PR DESCRIPTION
**Pull request by henrystivens**

### CODE REVIEW

File: app/views/_shared/templates/pdf.phtml [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+            </table>
+        </htmlpagefooter>
+    </body>
+</html>
+<?php
+$html = ob_get_clean();
+ob_clean();
+
+// Crea una instancia de la clase y le pasa el directorio temporal
+$mpdf = new Mpdf(['tempDir' => APP_PATH . 'temp/mpdf', 'margin-top' => 30]);
```
Comment (joanhey) [inline app/views/_shared/templates/pdf.phtml:52]: Separate variables to the beginning to easy configuration.
Like we have in the blog post.

**Review by joanhey — CHANGES_REQUESTED** [review]

File: app/config/databases.php [hunk 2]
```diff
@@ -10,6 +10,10 @@
         'username' => '',
         'password' => '',
     ],
+    'sandbox' => [
```
Comment (joanhey) [inline app/config/databases.php]: Change to default, to use the convention in new AR.

**Review by joanhey — CHANGES_REQUESTED** [review]

File: app/models/user.php [hunk 3]
```diff
@@ -5,5 +5,5 @@
  */
 class User extends \Kumbia\ActiveRecord\LiteRecord
 {
-    static protected $database = 'sandbox';
+    
 }
```
Comment (joanhey) [inline app/models/user.php]: Add a new line.

**Review by joanhey — CHANGES_REQUESTED** [review]

File: app/views/_shared/templates/pdf.phtml (hunk 1 above)
Comment (henrystivens) [reply app/views/_shared/templates/pdf.phtml:52]: Done

File: app/config/databases.php (hunk 2 above)
Comment (henrystivens) [reply app/config/databases.php]: Done.

File: app/models/user.php (hunk 3 above)
Comment (henrystivens) [reply app/models/user.php]: Done

**Review by joanhey — APPROVED** [review]

**Pull request merged**